metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -270,16 +270,32 @@ def _get_azure_ad_data(self, kind: str) -> Iterable[List]:
                 self.report.report_failure("_get_azure_ad_data_", error_str)
                 continue
 
+    def _map_identity_to_urn(self, func, id_to_extract, mapping_identifier, id_type):
+        result, error_str = None, None
+        try:
+            result = func(id_to_extract)
+        except Exception as e:
+            error_str = "Failed to extract DataHub {} from Azure AD {} with name {} due to '{}'".format(
```

Comment:
`due to '{}'"` is the `'{}'` intentional?

Reply:
yes. As that was a separate sentence coming from the exception so I added it.